src/borg/archiver.py

```diff
@@ -3509,6 +3521,8 @@ def define_borg_mount(parser):
         subparser.set_defaults(func=self.do_delete)
         subparser.add_argument('-n', '--dry-run', dest='dry_run', action='store_true',
                                help='do not change repository')
+        subparser.add_argument('--list', dest='output_list', action='store_true',
+                               help='output verbose list of archives or repositories')
```

Comment:
guess we do not have the repositories (plural) case, right? and "listing" just 1 does not make much sense, as this is obviously already known to the user.